### DISCUSSION TITLE
How to customize summary?

### DISCUSSION THREAD

**Discussion by LucienShui** [cotes2020/jekyll-theme-chirpy#1673] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want to use ChatGPT to summarize my blog, but I don't know where to place it, I tried `summary` and `description` field in _posts, but seems not working.

### Operations you have already tried

```markdown
title: foo
date: 2024-04-15 09:26:00 +0800
categories: [foo]
tags: [bar]
math: false
render_with_liquid: false
summary: 123
description: 123
```

### Anything else?

It seems there is an existing PR #1602, not sure for this requested feature. (I know about nothing except python)

Is there anyone know this?

**Comment by kungfux** [comment, marked as answer]
The possibility to specify post description manually you mentioned was implemented in #1602 as a new feature recently and was not released yet. I suppose it will be a part of the 7.0 release.

So, meanwhile you can keep custom descriptions and wait till the moment they start to work in the future once you upgrade to 7.0.

**Reply by LucienShui** [reply]
Thanks a lot, so according to the PR, I should use `description` in my post, is that right?

**Reply by kungfux** [reply]
That's correct.
